src/components/ModuleFields/Configurator/File.vue

```diff
@@ -45,6 +46,157 @@
         {{ $t('kind.file.view.showName') }}
       </b-form-checkbox>
     </b-form-group>
+
+    <template v-if="enablePreviewStyling">
+      <hr>
+
+      <h5 class="mb-2">
+        {{ $t('kind.file.view.previewStyle') }}
+      </h5>
+
+      <b-row
+        class="mb-2 mt-2"
+      >
+        <b-col
+          sm="12"
+          md="6"
+        >
+          <b-form-group
+            :label="$t('kind.file.view.height')"
+          >
+            <b-input-group
+              :append="$t('kind.file.view.px')"
+            >
+              <b-form-input
+                v-model="f.options.height"
+                type="number"
+                :number="true"
+                :placeholder="$t('kind.file.view.height')"
+              />
+            </b-input-group>
+          </b-form-group>
+        </b-col>
+        <b-col
+          sm="12"
+          md="6"
+          align-self="center"
```

Comment:
Instead of adding `align-self="center"` to every col. You can use the align-v property on the row